code/controllers/subsystem/tickets/SSmentor_tickets.dm

```diff
@@ -71,6 +72,22 @@ GLOBAL_REAL(SSmentor_tickets, /datum/controller/subsystem/tickets/mentor_tickets
 	if(message_key == null)
 		T.staffAssigned = null //if they cancel we dont need to hold this ticket anymore
 		return
+	if(message_key == "NCT Dispatch")
+		var/mob/living/carbon/human/trainee
+		var/mob/living/carbon/human/nct
+		trainee = get_mob_by_ckey(T.client_ckey)
```

Comment:
You should do this stuff after you've confirmed there is a NCT that can receive your message